app/src/main/java/com/example/oddhours/ui/home/HomeFragment.kt

```diff
@@ -7,28 +7,28 @@ import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import com.example.oddhours.R
-import com.example.oddhours.data.model.JobModel
 import com.example.oddhours.data.repository.JobRepository
-import com.example.oddhours.database.DatabaseHelper
 import kotlinx.android.synthetic.main.fragment_home.*
 
 class HomeFragment : Fragment() {
 
-//    private var layoutmanager: RecyclerView.LayoutManager? = null
-//    private var adapter: RecyclerView.Adapter<HomeAdapter.JobViewHolder>? = null
-    private var jobRepository: JobRepository = JobRepository()
+    private var layoutmanager: RecyclerView.LayoutManager? = null
+    private var adapter: RecyclerView.Adapter<HomeAdapter.JobViewHolder>? = null
+    private var jobRepository: JobRepository? = null
     private var hasJobs = false
 
     override fun onCreateView(
             inflater: LayoutInflater,
             container: ViewGroup?,
             savedInstanceState: Bundle?
     ): View? {
-        // retrieve jobs from DB
-        getAllJobs()
+        // retrieve jobs
+        jobRepository = JobRepository(requireActivity())
+        jobRepository!!.buildJobList()
 
```

Comment:
i think if we set `hasJobs = jobRepository!!.jobModelList?.isNotEmpty()` or something like that here, we won't need to repeat this line anywhere else we need it. i'm not sure but that was the idea behind having that variable initially